onnx2kerastl/reshape_layers.py

```diff
@@ -486,17 +487,26 @@ def convert_resize(node, params, layers, lambda_func, node_name, keras_name):
     else:
         raise Exception("unsupported resize method")
 
-    to_channel_last = keras.layers.Permute((2, 3, 1))(input_tensor)
+    axes = params.get('axes', list(range(input_tensor.shape.rank)))  # Default to resizing all axes
+
+    # Validate axes
+    rank = input_tensor.shape.rank
+    axes = [a if a >= 0 else a + rank for a in axes]  # Convert negative axes to positive
+    if any(a < 0 or a >= rank for a in axes):  # check that all axes values are within input rank
+        raise ValueError("Invalid axes value")
+
```

Comment:
I would add check if:
`K.image_data_format() == 'channels_first'`
and if not, raise NotImplementedError with reason.